frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -0,0 +1,17 @@
+import { AppRoute } from '~/libs/enums/enums.js';
+import { Navigate, useAppSelector } from '~/libs/hooks/hooks.js';
```

Comment:
This one `Navigate` is not a hook but a component - should be exported/imported from libs/components (look on how `RouterOutlet` was exported)